src/db/models/User.js

```diff
@@ -1,16 +1,26 @@
-export default (sequelize, DataTypes) => {
+module.exports = (sequelize, DataTypes) => {
   const User = sequelize.define('User', {
-    firstName: {
+    first_name: {
```

Comment:
I see you are using snake case style to name your columns, I am sure we agreed on using this only in the db. Please checkout [this link](http://docs.sequelizejs.com/manual/models-definition.html#configuration) to see how to set snake casing in the model definition. 
We should maintain pascal case style in our models, sequelize will transform it to snake case in the db after migration using the configuration we set above.

Reply:
I have corrected the case styling